packages/ai-constructs/src/conversation/runtime/types.ts

```diff
@@ -57,7 +57,17 @@ export type ConversationTurnEvent = {
       authorization: string;
     };
   };
-  messages: Array<ConversationMessage>;
+  /**
+   * @deprecated This field is going to be removed in upcoming releases.
+   */
+  messages?: Array<ConversationMessage>;
```

Comment:
I was going back and forth on this one and ultimately decided to keep backwards compatibility.

Factors that influenced this direction:
1. This is a change in protocol that requires upstream sender to adapt as well. It would disturb "custom handler scenario" in preview until upstream ships.
2. The alternative was to keep these changes on feature branch. But this seems to be the only change for GA that disturbs the protocol, and maintaining compatibility seems to be less effort than long running branch.
3. It's a drill of how we'd do this post GA.